### ISSUE TITLE
Add dialog titles for export dialogs

### ISSUE THREAD

**Issue by eloquence** [freedomofpress/securedrop-client#642]
Once #637 or a version if lands, we'll also need to update the titles in the dialogs to bring them more in line with the almost-spec here:

https://app.zeplin.io/project/5c807ea562f734bd2756b243/screen/5ddc3b8b9d9aab94be0542d5

This is separate from the dialog design, which will be broken out into its own issue.
